### ISSUE TITLE
incomplete description.

### ISSUE THREAD

**Issue by dagodax** [KaOS-Community-Packages/lumina-desktop#1]
Include in the description that has dependencies on kcp (fluxbox in this case).

**Comment by timcowchip** [comment]
No, fluxbox is in the depends line of PKGBUILD. I won't tell people where to find it, when they are already browsing KCP to discover lumina-desktop.
Give the KaOs users credit for some intelligence.

**Comment by timcowchip** [comment]
No, fluxbox is in the depends line of PKGBUILD. I won't tell people 
where to find it, when they are already browsing KCP to discover 
lumina-desktop. Give the KaOs users credit for some intelligence.

**Comment by dagodax** [comment]
From "Formatting standards for KCP" ( https://kaosx.us/kcp-guide/#Formatting_standards_for_KCP )

 *   Provide relative links in the readme to packages needed for your applications, which are not in the repositories, but need to be build from KCP also. And show the URL for the package.

Example for readme.md code, providing relative links:

    owncloud-client
    ===============

    Description:
    ownCloud is a software system for what is commonly termed cloud storage, for your personal Server.

    http://owncloud.org/

    Needed packages from KCP:

    Download and run makepkg -si in the package directory (with Dolphin and F4):

    [qtkeychain](../../../qtkeychain)

    [iniparser](../../../iniparser)

    [ocsync](../../../ocsync)

**Comment by timcowchip** [comment]
The example for owncloud-client is not the actual readme.md code 
own-client in KCP. The actual description for own-client does not 
include links to any other packages and so, neither will mine.

**Comment by dagodax** [comment]
That's because the current ownclient not require other kcp dependence. It's just an example. You want to break a standard, ok, you can feel rebellious.

**Comment by timcowchip** [comment]
A lightweight QT5 Desktop, originally developed by PC-BSD to run on top of fluxbox window manager (available in KCP)
